packages/react/src/SelectPanel/SelectPanel.features.stories.tsx

```diff
@@ -286,18 +278,14 @@ export const WithFooter = () => {
         placeholder="Select labels" // button text when no items are selected
         open={open}
         onOpenChange={setOpen}
-        items={selectedItemsSortedFirst}
+        items={filteredItems}
         selected={selected}
         onSelectedChange={setSelected}
         onFilterChange={setFilter}
         overlayProps={{width: 'small', height: 'medium'}}
-        footer={
-          <Button size="small" block>
-            Edit labels
-          </Button>
-        }
+        secondaryAction={<Button block>Edit labels</Button>}
```

Comment:
what does block do here? are we expecting consumers to always add that? 🤔

Reply:
It allows the button to expand, it is standard in other parts of the primer api too :/ I wish it wasn't tbh